src/main/resources/io/jenkins/plugins/pipelinegraphview/Messages_it.properties

```
@@ -22,11 +21,12 @@
 # THE SOFTWARE.
 
 startedAgo=Compilazione avviata {0} fa
+noBuilds=Nessuna build
 
 Util.millisecond={0} ms
 Util.second={0} s
 Util.minute={0} min
-Util.hour={0} h
-Util.day={0} g
-Util.month={0} m
-Util.year={0} a
+Util.hour  ={0} h
```

I made a mistake here.  This should be "o" for "ora" as the translation of "hour".
```suggestion
Util.hour  ={0} o
```